### ISSUE TITLE
Sang Staff Shows as 5 Tick

### ISSUE THREAD

**Issue by gainzzzz** [ngraves95/attacktimer#39]
Issue just started happening in a CM. When you attack the timer starts at 5 and counts down. Was working fine before today.

https://gyazo.com/fb1d58a4e6532f78c031b6c551ed912c

**Comment by gainzzzz** [comment]
Yep its broken, uninstalled then reinstalled plugin - restarted runelite - restarted computer - still shows as 5 tick.[https://gyazo.com/9081f32c25992777ec35b9afb0313d1f](url)

**Comment by calvin079** [comment]
having the same problem. sang staff shows up as 5 ticks

**Comment by Castro3halo** [comment]
Here to state the same issue, many charged staffs show as 5 tick when they should be 4 tick

**Comment by ngraves95** [comment]
Is this still an issue?
